content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -0,0 +1,152 @@
+# Integrating InfluxDB with Snowflake Using Apache Iceberg
+
+## Overview
+
+Snapshots for Snowflake enable users to export time-series data from InfluxDB into a structured format using Apache Iceberg. This integration facilitates efficient data sharing between InfluxDB and Snowflake without the need for complex ETL processes.
+
+### Key Benefits
+
+- **Efficient data access**: Query InfluxDB data directly from Snowflake.
+- **Cost-effective storage**: Optimize data retention and minimize storage costs.
+- **Supports AI and ML workloads**: Enhance machine learning applications by making time-series data accessible in Snowflake.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- A **Snowflake account** with necessary permissions.
+- Access to an **external object store** (such as AWS S3).
+- Familiarity with **Apache Iceberg** and **Snowflake**.
+
+## Step-by-step guide
+
+### Step 1: Configure external storage
+
+Set up an external storage location (such as AWS S3) to store Iceberg table data and metadata.
+
+#### Example: Configure an S3 stage in Snowflake
+
+```sql
+CREATE STAGE my_s3_stage 
+URL='s3://my-bucket/'
+STORAGE_INTEGRATION=my_storage_integration;
+```
+
+For more details, refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-catalog-integration-object-storage).
+
+### Step 2: Set up a catalog integration in Snowflake
+
+Set up a catalog integration in Snowflake to manage and load Iceberg tables efficiently.
+
+#### Example: Create a catalog integration in Snowflake
+
+```sql
+CREATE CATALOG INTEGRATION my_catalog_integration
+  CATALOG_SOURCE = 'OBJECT_STORE'
+  TABLE_FORMAT = 'ICEBERG'
+  ENABLED = TRUE;
+```
+
+For more details, refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-catalog-integration).
+
+### Step 3: Export InfluxDB data to Iceberg format
+
+Use InfluxDB's Iceberg Exporter to convert and export your time-series data into the Iceberg table format.
```

Comment:
```suggestion
Use the InfluxData's Iceberg exporter to convert and export your time-series data from your {{% product-name omit="Clustered" %}} cluster into the Iceberg table format.
```

Use the specific version name (`product-name` shortcode); the exporter isn't compatible with all versions (I assume). This becomes even more important for relying on AI to "deduce" feature support.